api/views/message_views.py

```
@@ -71,21 +78,56 @@ def post(self, request):
             ner_entities = user_message.ner_entities
             sentiment = user_message.sentiment
             overall_sentiment = user_message.overall_sentiment
             intent = user_message.intent
+
             #TODO: Call the chatbot model to get the response using the processed_text_use (Do all the processing you need, like NER, etc.)
+            previous_messages = Message.objects.filter(
+                chatbot=chatbot,
+                session=user_message.session
+            ).only("original_text", "sender", "timestamp").order_by('-timestamp')[:10]
+            conversation_context = [
+                {msg.sender: msg.original_text} 
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)